ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php

```diff
@@ -302,6 +320,7 @@ public function runServiceTask(ServiceTaskInterface $serviceTask, TokenInterface
             $token->setProperty('error', $error);
 
             // Log message errors
+            error_log('Service task failed: ' . $implementation . ' - ' . $exception->getMessage());
```

Comment:
please remove this line, in the next line is logged the same error...

Reply:
Removed